README.md

```diff
@@ -1,10 +1,18 @@
 Table Cell and Row Span
 =======================
 
+NOTE: This is a fork of an excellent idea implemented by @Neepawa, however
+the upstream seems to be abandoned for a while, so I collected different
+patches across all forks and also updated the extension to work with the
+latest version of [Python-Markdown][markdown].  I plan to maintain this
+version (and push changes to upstream) as long as I am running my personal
+blog on [Pelican][pelican], which heavily relies on Python-Markdown.
+
```

Comment:
This block should be deleted if the change is to be accepted since it is a part of my local fork's README.md.